src/aura/autocomplete/autocomplete.cmp

```diff
@@ -41,9 +41,9 @@
              data-typeahead="true">
             <div class="slds-form-element">
                 <aura:if isTrue="{!!empty(v.label)}">
-                    <label class="slds-form-element__label">{!v.label}</label>
+                    <span class="slds-form-element__label slds-float_left" id="inputLabel" >{!v.label}</span>
```

Comment:
Why span instead of label?

Reply:
<span> is recommended by the SLDS accessibility read only label. See [here](https://www.lightningdesignsystem.com/components/input/)